core/src/main/java/io/grpc/internal/BinaryLogProvider.java

```diff
@@ -172,8 +232,8 @@ public InputStream parse(InputStream stream) {
         return InternalClientInterceptors
             .wrapClientInterceptor(
```

Comment:
In a follow-up PR, get rid of this "helper" and integrate its contents directly into BinaryLog. Most of the implementation is just boilerplate and having it split makes this much more complicated.

Reply:
Agreed